README.md

```diff
@@ -19,35 +20,34 @@ You can follow these [steps](https://scalacenter.github.io/scala-advent-of-code/
 After you clone the repository, open a terminal and run:
 ```
 $ cd scala-advent-of-code
-$ scala-cli setup-ide src
-$ code .
+$ scala-cli setup-ide 2022
+$ code 2022
 ```
 
-`code .` will open Visual Studio Code and start Metals.
+`code 2022` will open Visual Studio Code and start Metals. If not you may have to go to the Metals pane and click
+the button labelled "Start Metals".
+
+When you navigate to a file, e.g. `2022/src/day01.scala` metals should index the project, and then display code lenses
+above each of the main methods `part1` and `part2`, as shown in this image:
+![](img/code-lenses.png)
```

Comment:
I cannot see the code lenses in the image. Am I blind or what? :laughing:

Reply:
yes you're right they were not there, oops